### ISSUE TITLE
Make the `kafka` PyPI namespace point to the `kafka-python` namespace

### ISSUE THREAD

**Issue by ghost** [dpkp/kafka-python#1726]
the latest version is submitted last year, and have a little bugs with python3.7, I find some mr for this bug is merged several month ago, so when submit a new version?

**Comment by dpkp** [comment]
Probably today or tomorrow!

**Comment by hh-h** [comment]
But what about pypi? It's still v1.3.5 https://pypi.org/project/kafka/

**Comment by dpkp** [comment]
Available at https://pypi.org/project/kafka-python/ (`pip install kafka-python`)

**Comment by hh-h** [comment]
Thanks for pointing me out. Probably, the other version (https://pypi.org/project/kafka/) need to be deleted from pypi, not to confuse people like me :)

**Comment by dpkp** [comment]
Deleting from pypi would likely cause builds, like yours, to break
unceremoniously. I'm not that mean :) Instead we are simply not updating
it, and letting your confusion lead to self-discovery. Hope the new release
treats you well!

**Comment by hh-h** [comment]
True, true. What about push small release with changed README file, that repository is obsolete and need to use the other one? :) 
![image](https://user-images.githubusercontent.com/8166963/54478462-7970c680-4823-11e9-812a-891bd47ab326.png)
Stable library only on 7th place...

**Comment by jeffwidman** [comment]
Agreed, it would be good to change the project description to say "Please use kafka-python instead as that is where we are releasing new versions."

**Comment by jeffwidman** [comment]
nudge @dpkp maybe you could update that Pypi project description per ☝️ ?

**Comment by wbarnha** [comment]
The easiest solution to this I see is building a workflow that publishes to kafka and kafka-python upon each release.
